readme.md

```diff
@@ -71,6 +71,14 @@ Returns an `Object` with the result, or `undefined` if there is no active window
 - `url` *(string?)* - URL of the active browser tab if the active window is Safari, Chrome, Edge, or Brave *(macOS only)*
 - `memoryUsage` *(number)* - Memory usage by the window owner process
 
+### activeWin.isAccessGranted()
+
+Check if the package has enough access to get the active window.
```

Comment:
This is not correct. It always has access, just not access to the window title, etc.